### PR TITLE
Sponsors 📯

### PR DESCRIPTION
**Pull request by egnwd**
Update the sponsor page to have grouped areas with all the new sponsors

### CODE REVIEW

**Comment by plietar** [comment]
Signature looks quite long. Do we really need the address and union logo ?

**Comment by plietar** [comment]
Sponsor page looks great

File: contents/data/sponsors.json [hunk 1]
```diff
@@ -1,92 +1,122 @@
...
+        "logo": "/img/sponsors/g_research_logo.png"
+      },
+      {
+        "name": "Netcraft",
+        "url": "https://www.netcraft.com",
+        "logo": "/img/sponsors/netcraft_logo.png"
+      },
+      {
+        "name": "NextJump",
+        "url": "http://www.nextjump.com",
```
Comment (plietar) [inline contents/data/sponsors.json]: Make that `https`

File: contents/data/sponsors.json [hunk 2]
```diff
@@ -1,92 +1,122 @@
...
+    "icon": "/img/icons/gold.png",
+    "sponsors": [
+      {
+        "name": "NewVoiceMedia",
+        "url": "https://www.newvoicemedia.com/",
+        "logo": "/img/sponsors/newvoicemedia_logo.png"
+      },
+      {
+        "name": "G-Research",
+        "url": "http://www.gresearch.co.uk",
```
Comment (plietar) [inline contents/data/sponsors.json]: `https://`

File: contents/data/sponsors.json [hunk 3]
```diff
@@ -1,92 +1,122 @@
...
+        "logo": "/img/sponsors/wluper_logo.png"
+      },
+      {
+        "name": "BlackRock",
+        "url": "https://www.blackrock.com/",
+        "logo": "/img/sponsors/blackrock_logo.png"
+      },
+      {
+        "name": "J.P. Morgan",
+        "url": "https://www.blackrock.com/",
```
Comment (plietar) [inline contents/data/sponsors.json]: Wrong link

File: contents/data/sponsors.json [hunk 4]
```diff
@@ -1,92 +1,122 @@
...
+        "logo": "/img/sponsors/sig_logo.png"
+      },
+      {
+        "name": "Bank of America Merrill Lynch",
+        "url": "https://www.bofaml.com",
+        "logo": "/img/sponsors/baml_logo.png"
+      },
+      {
+        "name": "Capital One",
+        "url": "http://www.capitalone.co.uk",
```
Comment (plietar) [inline contents/data/sponsors.json]: So the https website just redirect to http, so I don't really care what we use.

Comment (egnwd) [reply contents/data/sponsors.json]: I'll add it just in case they start not redirecting

**Pull request merged**